### PR TITLE
Move AMM price to tooltip

### PR DESCRIPTION
**Pull request by aeolianeth**
## What does this PR do and why?

Related to https://github.com/jbx-protocol/juice-interface/issues/344, this PR moves AMM price display to tooltip in preparation for multiple AMMs (sushi is next).

## Screenshots or screen recordings

| light | dark |
| --- | --- |
| Screen Shot 2022-01-17 at 10 28 30 pm | Screen Shot 2022-01-17 at 10 28 19 pm |

| state | tooltip |
| --- | --- |
| has price | Screen Shot 2022-01-17 at 10 23 00 pm |
| no sushi (for a few days at least) | Screen Shot 2022-01-17 at 10 23 18 pm |
| No LP | Screen Shot 2022-01-17 at 10 23 42 pm |
| No ERC20 | Screen Shot 2022-01-17 at 10 24 02 pm |

## Acceptance checklist

- [x] I have evaluated the [Approval Guidelines](../../CONTRIBUTING.md#approval-guidelines) for this PR.
- [x] I have tested this PR in [all supported browsers](../../CONTRIBUTING.md#browser-support).

### CODE REVIEW

**Comment by mejango** [comment]
Can we land this?

**Comment by aeolianeth** [comment]
> Can we land this?

LFG.

@peripheralist mind reviewing this when you get a chance? Currently AFK so haven't cross browser tested - if this can wait until tomorrow, I can test it, but if not, feel free to take it for a spin now and merge if all good.

**Comment by aeolianeth** [comment]
> > Can we land this?
> 
> LFG.
> 
> Peri mind reviewing this when you get a chance? Currently AFK so haven't cross browser tested - if this can wait until tomorrow, I can test it, but if not, feel free to take it for a spin now and merge if all good.

Testing now, will shoot back your way when ready.

**Comment by aeolianeth** [comment]
@peripheralist okay! Tested, modified, and ready for your review! ❤️

File: src/components/Dashboard/ProjectActivity/RichNote.tsx [hunk 1]
```diff
@@ -47,7 +47,7 @@ export default function RichNote({
       : note
 
   return (
-    <p style={{ marginTop: 5, ...style }}>
+    <div style={{ marginTop: 5, ...style }}>
```
Comment (aeolianeth) [inline src/components/Dashboard/ProjectActivity/RichNote.tsx:50]: Fixes a console warning about `div`s existing inside `p`. No user-facing changes AFAICT

File: src/components/Navbar/MenuItems.tsx [hunk 2]
```diff
@@ -125,20 +125,24 @@ export const menu = (onClickMenuItems?: VoidFunction) => {
         heading="Resources"
         dropdownItems={[
           <DropdownItem
+            key="docs"
```
Comment (aeolianeth) [inline src/components/Navbar/MenuItems.tsx:128]: Fixes console warning about lack-of keys

Comment (peripheralist) [reply src/components/Navbar/MenuItems.tsx:128]: awesome

**Review by peripheralist — APPROVED** [review]
Super solid work, everything looks in order to me. lfg!

**Pull request merged**